hmp/io.py

```diff
@@ -460,6 +460,10 @@ def _epoch_selection(epochs,  # noqa: PLR0912, PLR0913
             "Metadata should be a pandas data-frame as generated by mne or be contained "
             "in the passed epoch data"
         )
+    if 'block' in metadata_i.columns:
+        metadata_i = metadata_i.drop(columns=['block'])
+    if 'Block' in metadata_i.columns:
+        metadata_i = metadata_i.drop(columns=['Block'])
```

Comment:
That' very specific, we might want to have some column mapping instead, I'll think ahout it